src/functions.js

```diff
@@ -0,0 +1,38 @@
+const { gameOfThronesEpisodes } = require("./data/gameOfThronesData");
+/**
+ *
+ * @param {string} searchWord
+ * @returns {any[]}
+ */
+function filterThroughEpisodes(searchWord) {
+    return gameOfThronesEpisodes.filter((episode) => {
+        const nameMatch = episode.name
+            .toLowerCase()
+            .includes(searchWord.toLowerCase());
+        const summaryMatch = episode.summary
+            .toLowerCase()
+            .includes(searchWord.toLowerCase());
+
+        return nameMatch || summaryMatch;
```

Comment:
cool function, really interesting way to do this! never seen this type of return done before